### PR TITLE
Pp 628 implement default and custom bottom bar navigation

### PR DESCRIPTION
**Pull request by mrkulik**
Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-07-10 at 10 46 45
Simulator Screenshot - iPhone SE (1st generation) - 2024-07-10 at 10 47 53
Simulator Screenshot - iPhone SE (3rd generation) - 2024-07-10 at 10 46 43

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/DefaultSkontoBottomNavigationBar.swift [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+final class DefaultSkontoBottomNavigationBar: UIView {
+    private lazy var configuration = GiniBankConfiguration.shared
+
+    private lazy var proceedButton: MultilineTitleButton = {
+        let button = MultilineTitleButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.configure(with: configuration.primaryButtonConfiguration)
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        let title = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.paybutton.title",
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/DefaultSkontoBottomNavigationBar.swift:18]: [suggestion]: Please use the button text from Skonto.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/DefaultSkontoBottomNavigationBar.swift [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    }()
+
+    private lazy var totalLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = configuration.textStyleFonts[.body]
+        label.textColor = .giniColorScheme().text.primary.uiColor()
+        label.adjustsFontForContentSizeCategory = true
+        label.setContentHuggingPriority(.required, for: .vertical)
+        let text = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.lineitem.totalpricetitle",
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/DefaultSkontoBottomNavigationBar.swift:63]: [suggestion] Please use the texts from Skonto.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift [hunk 3]
```diff
@@ -192,6 +192,18 @@ public final class GiniBarButton {
...
+        stackView.setContentHuggingPriority(priority, for: axis)
+        titleLabel.setContentHuggingPriority(priority, for: axis)
+        imageView.setContentHuggingPriority(priority, for: axis)
+    }
+
+    public func setContentCompressionResistancePriority(_ priority: UILayoutPriority, for axis: NSLayoutConstraint.Axis) {
+        stackView.setContentCompressionResistancePriority(priority, for: axis)
+        titleLabel.setContentCompressionResistancePriority(priority, for: axis)
+        imageView.setContentCompressionResistancePriority(priority, for: axis)
+    }
```
Comment (ValentinaIancu-Gini) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift:206]: [clarification] We need to make sure we did not break anything related to GiniBarButton by adding this.

**Review by ValentinaIancu-Gini — COMMENTED** [review]
@mrkulik Grate job!

Please see my suggestions.

As well I found some small UI issues.

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift (hunk 3 above)
Comment (mrkulik) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift:206]: All should be good, I have done it like methods to update only where it's needed. So we will use it only on Skonto for dynamic layout

**Comment by ValentinaIancu-Gini** [comment]
@mrkulik Looks good. 
I left some minor suggestions.

Also now that we hide the Help button the bottom navigation bar is looking awkward. Maybe is good to insert a spacer to the right of the Proceed button until the next phase when we add the Help button. What do you say?

**Review by ValentinaIancu-Gini — APPROVED** [review]
@mrkulik Looks good. Thank you!

I will wait for all the checks to pass and I can give a build for Vlad to test this pr changes.

**Pull request merged**